src/js/media-chrome-button.js

```diff
@@ -4,37 +4,33 @@ import { window, document } from './utils/server-safe-globals.js';
 
 const template = document.createElement('template');
 
-template.innerHTML = `
+template.innerHTML = /*html*/`
 <style>
   :host {
+    font: var(--media-font,
```

Comment:
I think I would still slightly prefer individual properties over the font shorthand. It's more explicit and would help with searchability, since you could find it via grepping for say `line-height`.

Reply:
cool will update. I don't feel so strongly about it

Reply:
oh actually then we can't use `--media-font` as a CSS var for setting all at once which includes some more things like 

[font-stretch](https://developer.mozilla.org/en-US/docs/Web/CSS/font-stretch)
[font-style](https://developer.mozilla.org/en-US/docs/Web/CSS/font-style)
[font-variant](https://developer.mozilla.org/en-US/docs/Web/CSS/font-variant)

I think that extra function and it taking less CSS makes it a better choice. what do you think?

Reply:
Theoretically, we could do some fancy var stuff to unset the other properties if --media-font is set, but probably too much work. Is --media-font needed?
While, I do think the individual properties are a bit better, I'm OK with keeping it as the shorthand.

Reply:
> Is --media-font needed?

it would give devs the full flexibility of all font css props if needed with small addition in code size. 
seems like a good devex